File: MODUL 4/Soal 4/PRAK404-2410817210026-Fadhil_Syahdama_Mahatma_Putra.py
def main():
    while True:

        print("\nPilih program")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Exit")
        zani = int(input("Masukkan pilihan : "))

        if zani == 1:
            satu = float(input("Masukkan nilai pertama : "))
            duwa = float(input("Masukkan nilai kedua : "))
            print(f"Hasil penjumlahan antara {satu:.2f} dengan {duwa:.2f} adalah {satu + duwa:.2f}")

        elif zani == 2:
            satu = float(input("Masukkan nilai pertama : "))
            duwa = float(input("Masukkan nilai kedua : "))
            print(f"Hasil pengurangan antara {satu:.2f} dengan {duwa:.2f} adalah {satu - duwa:.2f}")

        elif zani == 3 :
            satu = float(input("Masukkan nilai pertama : "))
            duwa = float(input("Masukkan nilai kedua : "))
            print(f"Hasil perkalian antara {satu:.2f} dengan {duwa:.2f} adalah {satu * duwa:.2f}")

        elif zani == 4 :
            satu = float(input("Masukkan nilai pertama : "))
            duwa = float(input("Masukkan nilai kedua : "))
            print(f"Hasil pembagian antara {satu:.2f} dengan {duwa:.2f} adalah {satu / duwa:.2f}")

        elif zani == 5:
            print("Terimakasih, telah menggunakan kalkulator Fadhil Syahdama Mahatma Putra")
            break
        
        else :
            print("Input anda salah, silahkan coba lagi ")


main()

File: MODUL 4/Soal 4/PRAK404-2410817210026-Fadhil_Syahdama_Mahatma_Putra.c
#include <stdio.h>
int main(){

    int zani;
    float satu, duwa;

    do{
    printf("\nPilih program\n");
    printf("1. Penjumlahan\n"); 
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");
    printf("Masukkan Pilihan : ");
    scanf("%d", &zani);

    switch (zani)
    {
    case 1:
        printf("Masukkan nilai pertama : ");
        scanf("%f", &satu);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &duwa);
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", satu, duwa, satu + duwa);
        break;

    case 2 :
        printf("Masukkan nilai pertama : ");
        scanf("%f", &satu);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &duwa);
        printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", satu, duwa, satu-duwa);
        break;

    case 3 :
        printf("Masukkan nilai pertama : ");
        scanf("%f", &satu);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &duwa);
        printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", satu, duwa, satu*duwa);
        break;

    case 4 :
        printf("Masukkan nilai pertama : ");
        scanf("%f", &satu);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &duwa);
        printf("Hasil dari pembagian %.2f dengan %.2f adalah %.2f\n", satu, duwa, satu/duwa);\
        break;
    
    case 5 :
        printf("Terimakasih, telah menggunakan kalkulator Fadhil Syahdama Mahatma Putra\n");
        break;
    
    default:
        printf("Input anda salah, silahkan coba lagi\n");
        break;
    }
    } while (zani != 5);
}